### PR TITLE
enable aarch64 linux builds

### PR DESCRIPTION
**Pull request by blowekamp**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ conda_forge_output_validation: true
 build_platform:
   osx_arm64: osx_64
 linux_aarch64:
-  enabled: false
+  enabled: trie
```
Comment (zivy) [inline conda-forge.yml]: trie -> true

**Review by zivy — CHANGES_REQUESTED** [review]

**Review by zivy — APPROVED** [review]

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

**Comment by blowekamp** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,6 +28,7 @@ requirements:
     - swig    4.0.*
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
 
   host:
     - libitk-devel  5.3.*
```
Comment (hmaarrfk) [inline recipe/meta.yaml:34]: can we try to unpin this?

**Comment by blowekamp** [comment]
closing for  #31

**Pull request closed without merging**